strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxEntityManager.java

```diff
@@ -202,12 +207,37 @@ public Query createNativeQuery(String sqlString,
         throw new UnsupportedOperationException("createNativeQuery");
     }
 
+    @Override
+    public StoredProcedureQuery createNamedStoredProcedureQuery(String s) {
+        return null;
+    }
+
+    @Override
+    public StoredProcedureQuery createStoredProcedureQuery(String s) {
```

Comment:
I'm assuming these methods are required by some interface... Shouldn't we have a proper implementation for them?

@epidemia , @sbespalov : What do you think?